lib/src/api/opt/resource.rs

```diff
@@ -104,7 +96,7 @@ impl From<&Edges> for Resource {
 
 impl From<&str> for Resource {
 	fn from(s: &str) -> Self {
-		match sql::thing(s) {
+		match syn::thing(s) {
```

Comment:
I wouldn't expect this change to be necessary. Does this mean `surrealdb::sql::thing` is no longer available @DelSkayn?

Reply:
The parser functions are reexported under `sql` to prevent breaking changes but for consistency I changed most usages of `sql::parser` to `syn::parser`

Reply:
I think it makes it more clear what the function does.